app/Http/Controllers/WikiEntityImportController.php

```diff
@@ -9,9 +9,27 @@
 use App\WikiEntityImport;
 use App\Jobs\WikiEntityImportJob;
 use Carbon\Carbon;
+use Prometheus\CollectorRegistry;
+use Prometheus\Counter;
 
 class WikiEntityImportController extends Controller
 {
+    private Counter $successfulCounter;
+    private Counter $failedCounter;
+
+    public function __construct(CollectorRegistry $registry)
+    {
+        $this->successfulCounter = $registry->getOrRegisterCounter(
```

Comment:
There's a tricky decision here: if we define these counters only once, it means they will be missing from the metrics if they have never been increased yet (the controller that answers the scraper will not know about it yet as it's never been persisted).

Alternatively we could:
- duplicate this definition in the `Metrics/WikiEntityImport` class
- move all metrics definitions to the service provider and have stub `metrics` methods on the metrics classes themselves

Reply:
If I understand right this isn't really a big problem? It just means that instead of `0` we will have `null` until the first import?